lldb/source/Plugins/Process/FreeBSD/NativeProcessFreeBSD.cpp

```diff
@@ -48,14 +48,37 @@ static Status EnsureFDFlags(int fd, int flags) {
   return error;
 }
 
+static Status CanTrace() {
+  Status status;
+  int proc_debug, ret;
+  size_t len = sizeof(proc_debug);
+  ret = ::sysctlbyname("security.bsd.unprivileged_proc_debug", &proc_debug,
+                       &len, nullptr, 0);
+  if (ret != 0)
+    return Status("sysctlbyname() security.bsd.unprivileged_proc_debug failed");
```

Comment:
How long has FreeBSD had `security.bsd.unprivileged_proc_debug`? If you're debugging an older FreeBSD system that does not have it, this will prevent it from debugging anything.

Reply:
since a long time :) e.g. even 9.0, EOL since more than a decade, had it.

Reply:
It's had the current name for over 2 decades, https://github.com/freebsd/freebsd-src/commit/e409590d0e0e57c6ec37d95bcb9fa3728051ebb1

The sysctl existed prior to that with a different name; I didn't bother looking to see when that was introduced.

Reply:
Thanks for the explanation. Consider my concerns addressed, don't let my "Requested Changes" block this from going in after another approval. :)